frontend/public/components/monitoring/query-browser.tsx

```diff
@@ -70,6 +70,8 @@ import { PrometheusAPIError } from './types';
 const spans = ['5m', '15m', '30m', '1h', '2h', '6h', '12h', '1d', '2d', '1w', '2w'];
 export const colors = queryBrowserTheme.line.colorScale;
 
+const SafeVictoryPortal = (VictoryPortal as unknown) as React.ComponentType;
```

Comment:
@jgbernalp Could you please add a code comment explaining what this is for?

Reply:
@kyoto Sorry, forgot to put the WIP. I wanted to run the test to track the root issue.

Reply:
👍